### PR TITLE
Refacto menu + mod-vertical + scrollBox

### PR DESCRIPTION
**Pull request by vvalentin-lucca**
## Description

- [x] Deactivated `a` are now without `href`.
- [x] `mod-S` is now automatically applied to any embedded digital badges. 
- [x] Modifiers and states are now managed with CSS variables.
- [x] A `mod-vertical` has been added.
- [x] A new wrapper component (`.scrollBox`) now displays a scrollbar and shadows when necessary. 

-----

-----

Capture d’écran 2024-02-22 à 17 27 09

Capture d’écran 2024-02-22 à 17 26 54

### CODE REVIEW

**Comment by LuccaIntegration** [comment]
:woman_cook: https://lucca-front.lucca.tech/PR-2565/storybook

**Comment by CCNET-iLucca** [comment]
[Tests d'interfaces](https://qa.lucca.tech/13211/ui-diff)

**Comment by LuccaIntegration** [comment]
:woman_cook: https://lucca-front.lucca.tech/PR-2565/storybook

**Comment by CCNET-iLucca** [comment]
[Tests d'interfaces](https://qa.lucca.tech/13254/ui-diff)

**Comment by LuccaIntegration** [comment]
:woman_cook: https://lucca-front.lucca.tech/PR-2565/storybook

**Comment by CCNET-iLucca** [comment]
[Tests d'interfaces](https://qa.lucca.tech/13257/ui-diff)

**Comment by LuccaIntegration** [comment]
:woman_cook: https://lucca-front.lucca.tech/PR-2565/storybook

**Comment by LuccaIntegration** [comment]
:woman_cook: https://lucca-front.lucca.tech/PR-2565/storybook

**Comment by LuccaIntegration** [comment]
:woman_cook: https://lucca-front.lucca.tech/PR-2565/storybook

**Comment by CCNET-iLucca** [comment]
[Tests d'interfaces](https://qa.lucca.tech/13271/ui-diff)

**Comment by LuccaIntegration** [comment]
:woman_cook: https://lucca-front.lucca.tech/PR-2565/storybook

**Comment by CCNET-iLucca** [comment]
[Tests d'interfaces](https://qa.lucca.tech/13321/ui-diff)

File: stories/documentation/navigation/menu/menu-basic.stories.ts [hunk 1]
```diff
@@ -30,30 +31,58 @@ export default {
...
+			</li>
+			<li class="menu-list-item">
+				<a href="#" class="menu-list-item-action">
+					Menu 2
+				</a>
+			</li>
+			<li class="menu-list-item">
+				<span class="menu-list-item-action is-disabled">
+					Menu 3
+				</span>
```
Comment (fbasmaison-lucca) [inline stories/documentation/navigation/menu/menu-basic.stories.ts:64]: Par curiosité : Pourquoi pas uniquement virer le `href` ? Pour ne pas hériter de certains styles ?

**Comment by LuccaIntegration** [comment]
:woman_cook: https://lucca-front.lucca.tech/PR-2565/storybook

**Comment by CCNET-iLucca** [comment]
[Tests d'interfaces](https://qa.lucca.tech/13657/ui-diff)

File: stories/documentation/navigation/menu/menu-basic.stories.ts (hunk 1 above)
Comment (MarcFairbrother) [reply stories/documentation/navigation/menu/menu-basic.stories.ts]: I'm afraid including anchor tags without the `href` attribute in the docs will be confusing for some users.

Comment (vvalentin-lucca) [reply stories/documentation/navigation/menu/menu-basic.stories.ts]: This can be documented if required. Both work for me.

**Comment by LuccaIntegration** [comment]
:woman_cook: https://lucca-front.lucca.tech/PR-2565/storybook

**Comment by CCNET-iLucca** [comment]
[Tests d'interfaces](https://qa.lucca.tech/15714/ui-diff)

**Comment by LuccaIntegration** [comment]
:woman_cook: https://lucca-front.lucca.tech/PR-2565/storybook

**Comment by CCNET-iLucca** [comment]
[Tests d'interfaces](https://qa.lucca.tech/17346/ui-diff)

**Comment by LuccaIntegration** [comment]
:woman_cook: https://lucca-front.lucca.tech/PR-2565/storybook

**Comment by CCNET-iLucca** [comment]
[Tests d'interfaces](https://qa.lucca.tech/17723/ui-diff)

**Comment by LuccaIntegration** [comment]
:woman_cook: https://lucca-front.lucca.tech/PR-2565/storybook

**Comment by LuccaIntegration** [comment]
:woman_cook: https://lucca-front.lucca.tech/PR-2565/storybook

**Comment by CCNET-iLucca** [comment]
[Tests d'interfaces](https://qa.lucca.tech/17727/ui-diff)

**Comment by LuccaIntegration** [comment]
:woman_cook: https://lucca-front.lucca.tech/PR-2565/storybook

**Comment by CCNET-iLucca** [comment]
[Tests d'interfaces](https://qa.lucca.tech/17728/ui-diff)

**Comment by LuccaIntegration** [comment]
:woman_cook: https://lucca-front.lucca.tech/PR-2565/storybook

**Comment by CCNET-iLucca** [comment]
[Tests d'interfaces](https://qa.lucca.tech/17743/ui-diff)

**Review by jeremie-lucca — APPROVED** [review]

**Comment by LuccaIntegration** [comment]
:woman_cook: https://lucca-front.lucca.tech/PR-2565/storybook

**Comment by CCNET-iLucca** [comment]
[Tests d'interfaces](https://qa.lucca.tech/17750/ui-diff)

**Pull request merged**